server/server_other.go

```diff
@@ -8,56 +8,48 @@
 package server
 
 import (
-	"fmt"
```

Comment:
I'm unsure of the point of this file? Is it for Darwin? Will it fool people into thinking things work? Just trying to understand.